### ISSUE TITLE
Small changes needed to create Python 3 compatable version

### ISSUE THREAD

**Issue by timothyja** [astrofrog/idlsave#6]
1. Change all <> to !=
2. Add () to print statements

**Comment by timothyja** [comment]
There was also some issues with binary/string compares and the **init** import. For my python 3 versions see: http://www.itsqueeze.com/2012/04/read-idl-sav-files-into-python-3/

**Comment by astrofrog** [comment]
Thanks for the report! I think the developer version of idlsave should be Python-3 compatible already:

```
git clone git://github.com/astrofrog/idlsave.git
cd idlsave
python setup.py install
```

Let me know if it works, and I'll release a new stable version!

**Comment by timothyja** [comment]
Works fine for me.

**Comment by timothyja** [comment]
Also just read this in the readme file "IDLSave has been tested with a number of files generated by IDL versions 5
  to 7"

Just FYI I've been reading in files created with IDL 8.1

**Comment by astrofrog** [comment]
I updated the README to mention IDL 8 - thanks! (fixed in 468dccf184842b646be9364e1c7a32cf4f78b4c2)
